src/Database/DatabaseServiceProvider.php

```diff
@@ -84,7 +84,7 @@ protected function getDefaultDatabaseDriver()
      */
     protected function swapSchemaBuilderBlueprint()
     {
-        $this->app['events']->listen('db.schema.getBuilder', function ($builder) {
+        $this->app['events']->listen('db.schema.getBuilder', function (\Illuminate\Database\Schema\Builder $builder) {
```

Comment:
What's the benefit of this?

Reply:
Typehint for an ide, so we don't get errors in this code